test/distributed/tensor/parallel/test_fsdp_2d_parallel.py

```diff
@@ -246,9 +249,11 @@ def test_2d_state_dict(self, is_even_sharded_model):
         )
         tp_mesh = mesh_2d["tp"]
         dp_mesh = mesh_2d["dp"]
-        model_2d = parallelize_module(
-            simple_model().cuda(), tp_mesh, PairwiseParallel()
-        )
+        parallelize_plan = {
```

Comment:
looks like there's a legit CI failure that's related to `SimpleModelUneven` where in the `SimpleModelUneven`  case it defined nn.Sequential instead of nn.Module, I think we should change the `SimpleModelUneven`  to define the same as `SimpleModel` instead of using `nn.Sequential` as we just want to test the Linear size to be unevenly divided, not testing nn.Sequential

Reply:
@wz337 Iris told me the tests would still fail even if we make such changes, because of potential size mismatch. I cannot verify locally because of devgpu issues.

Per Iris's suggestion, I'm disabling all three tests involving the sequential SimpleModelUneven for now to see if tests from ciflow/periodic can pass so that I can land. Meanwhile I'll be working on fixing devgpu.

Reply:
Yes, for `SimpleModelUneven` it is an no-op for currently TP API if using `PairwiseParallel()` that's why test passed previously.